src/main/java/uk/gov/companieshouse/itemhandler/logging/LoggingUtils.java

```diff
@@ -27,6 +27,7 @@ private LoggingUtils() {
     public static final String ORDER_RECEIVED_GROUP_ERROR = "order_received_error";
     public static final String ORDER_REFERENCE_NUMBER = "order_reference_number";
     public static final String ORDER_URI = "order_uri";
+    public static final String DESCRIPTION_LOG_KEY = "description_key";
 
```

Comment:
Is this `description_key` or just `description`?

Reply:
Yes I will change this to description

Reply:
OK maybe not. I suppose it may be easier to understand at a glance in this case if all the keys were declared here in the `LoggingUtils` class?